src/main.py

```diff
@@ -80,11 +80,29 @@ async def on_message(self, message) -> None:
         # Ignore bot accounts
         if message.author.bot:
             return
+        
+        # word blacklist
+        text_lowercase = message.content.lower()
+        # Iterate through all words in the blacklist
+        for word in configuration.WORDS_CENSORED:
+            if word in text_lowercase:
```

Comment:
```suggestion
            # Make word lowercase to avoid always false match on listed words with capitalized letters
            if word.lower() in text_lowercase:
```